app/sbna-poc/v1_0__initial-form/fields/finance.ts

```diff
@@ -305,7 +276,7 @@ export const baseFinanceFields: Array<FormWizard.Field> = [
   {
     text: 'Is [subject] affected by gambling?',
     code: 'finance_gambling',
-    type: FieldType.Radio,
+    type: FieldType.CheckBox,
```

Comment:
When using `FieldType.CheckBox` we need to add `multiple: true` to the field configuration to allow for an array of values